docs/StyleGuide.md

```diff
@@ -1046,9 +977,9 @@ the rules below, it is good practice to update the code's style to match them.
 
 18. __Records__:
 
-    For records we often use `NamedFieldPuns` to make it convenient to
-    extract fields from the record. We also tend to use `RecordWildCards`
-    when the fields of the record are all of the form
+    For records we often use `NamedFieldPuns` to make it convenient to extract
+    fields from the record. We use the following convention when naming fields
+    to avoid duplicate record fields (we do not use `DuplicateRecordFields`):
 
     ```haskell
     data SomeRecord = SomeRecord {
```

Comment:
There are actually a lot of places where we use an `srSomeField` naming convention, so mayb we should reconsider this section?

Reply:
In certain circumstances I prefer not prefixing the accessors if the module is intended to be imported qualified. But for this particular case perhaps we should mention that using the first letters of `SomeRecord` is also an option (for instance when the type name is too long.

Reply:
Done